app-template/src/Navbar.tsx

```diff
@@ -157,86 +109,73 @@ export class Navbar extends React.Component<{}, NavbarState> {
-                          onClick={toggleUserMenu}
-                          className="dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content"
-                        >
-                          Teams
-                        </a>
-                        <a
-                          href={`https://www.synapse.org/#!Profile:${userprofile.ownerId}/challenges`}
-                          onClick={toggleUserMenu}
-                          className="dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content"
-                        >
-                          Challenges
-                        </a>
-                        <a
-                          href={`https://www.synapse.org/#!Profile:${userprofile.ownerId}/downloads`}
-                          onClick={toggleUserMenu}
-                          className="dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content border-bottom-1"
-                        >
-                          Downloads
-                        </a>
-                        <a
-                          href={`https://www.synapse.org/#!Profile:${userprofile.ownerId}/settings`}
-                          onClick={toggleUserMenu}
-                          className="dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content border-bottom-1"
-                        >
-                          Settings
-                        </a>
-                        <button
-                          // @ts-ignore
-                          onClick={() => signOut()}
-                          role="button"
-                          className="dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content border-bottom-1 hand-cursor"
-                        >
-                          Sign Out
-                        </button>
-                      </div>
-                    }
-                    </div>
+                <Dropdown>
+                    <Dropdown.Toggle variant="light" id="user-menu-button">
+                        <UserCard
+                          userProfile={userprofile}
+                          size={SynapseConstants.SMALL_USER_CARD}
+                          preSignedURL={userprofile.clientPreSignedURL}
+                          hideText={true}
+                          link="javascript:void(0)"
+                        />
+                        <SvgIcon>
+                          {
+                            // Material expand more svg https://material.io/tools/icons/?icon=expand_more&style=baseline
+                          }
+                          <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z" />
+                        </SvgIcon>
+                  </Dropdown.Toggle>
+                  <Dropdown.Menu className="user-menu-dropdown portal-nav-menu">
+                      <Dropdown.Item
```

Comment:
just wondering shy we are defining the items list explicitly vs creating an array of hrefs and using a map

Reply:
good point! I hadn't though of that, changed that now